frontend/integration-tests/views/catalog.view.ts

```diff
@@ -1,8 +1,12 @@
-import { $, $$, browser, ExpectedConditions as until } from 'protractor';
+import { $, $$, browser, by, ExpectedConditions as until } from 'protractor';
 
 export const categoryTabs = $$('.vertical-tabs-pf-tab > a');
 export const pageHeading = $('.co-catalog-page__heading');
 export const pageNumberItemsHeading = $('.co-catalog-page__num-items');
 export const pageHeadingNumberOfItems = () => pageNumberItemsHeading.getText()
   .then(text => parseInt(text.substring(0, text.indexOf(' items')), 10));
 export const catalogDetailsLoaded = () => browser.wait(until.presenceOf($('.modal-content')), 10000).then(() => browser.sleep(1000));
+export const createServiceInstanceButton = $('.co-catalog-page__overlay-create');
+export const createServiceInstanceFormIsLoaded = () => browser.wait(until.presenceOf($('.co-create-service-instance')), 10000).then(() => browser.sleep(500));
```

Comment:
I think we'll want to pick up some of the fixes from #1160 to determine when the form is done loading. @TheRealJon added some utilities to make sure there are no loaders present anywhere on the page.

Reply:
switched to 'no loaders' check